core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -255,7 +255,7 @@ object Fiber {
   /**
    * Returns a `Fiber` that is backed by the specified `Future`.
    */
-  final def fromFuture[A](thunk: => Future[A]): Fiber[Throwable, A] =
+  final def fromFuture[A](thunk: Future[A]): Fiber[Throwable, A] =
```

Comment:
I do not like this change. Many `from*` methods are lazy. If the user chooses to put effects there, that's their business, not ours. I know that contradicts #527.